examples/event-sourced-domain-with-tests/Invites.php

```diff
@@ -34,7 +34,7 @@ public static function invite($invitationId, $name)
      *
      * {@inheritdoc}
      */
-    public function getAggregateRootId()
+    public function getAggregateRootId() : string
```

Comment:
apparently the CS fixers fails. Can you remove the space before the colon? "public function getAggregateRootId(): string"

Reply:
Done, as well as added the examples folder to phpunit.xml.dist